docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+services:
+
+  db:
+    image: postgres:9.5-alpine
+    container_name: "kryla_api_db"
+    volumes:
+      - kryla_api_database-postgres:/var/lib/postgresql/data
+
+  backend:
+    container_name: "kryla_api_rails"
+    build: .
+    command: /bin/sh -c "rm -f /myapp/tmp/pids/server.pid && bundle exec rails s -p 3000 -b 0.0.0.0"
+    volumes:
+      - .:/myapp
+    ports:
+      - 3000:3000
+    depends_on:
+      - db
+      - redis
+    env_file:
+      - .env
+
+  redis:
+    container_name: kryla_api_redis
+    image: redis:4
+    ports:
+      - '6379:6379'
```

Comment:
I would prefer to use double quotes in all .yml files.

Reply:
Switched to double quotes.